docs/core/deploying/creating-nuget-packages.md

```diff
@@ -4,12 +4,7 @@ description: Learn how to create a NuGet package with the 'dotnet pack' command.
 author: cartermp
 ms.author: mairaw
 ms.date: 06/20/2016
-ms.topic: conceptual
-ms.prod: dotnet-core
 ms.technology: dotnet-cli
```

Comment:
Should this one stay?

Reply:
Yes, they are not on the global settings so I left them. @rpetrusha do you think these should continue to be tagged as CLI?